src/apps/umami/arbitrum/umami.marinate.token-fetcher.ts

```diff
@@ -43,12 +70,22 @@ export class ArbitrumUmamiMarinateTokenFetcher implements PositionFetcher<AppTok
     const baseTokenDependencies = await this.appToolkit.getBaseTokenPrices(network);
     const underlyingToken = baseTokenDependencies.find(v => v.address === UMAMI_ADDRESS);
     if (!underlyingToken) return [];
+
+    const { apr, marinateTVL } = await this.getUmamiInformations();
+
     const tokens = [underlyingToken];
     const pricePerShare = 1.0;
     const price = pricePerShare * underlyingToken.price;
     const label = `Marinating UMAMI`;
     const images = getImagesFromToken(underlyingToken);
     const secondaryLabel = buildDollarDisplayItem(price);
+    const tertiaryLabel = `${apr}% APR`;
+    const statsItems = [
+      {
+        label: 'TVL',
+        value: `${marinateTVL}`,
+      },
+    ];
```

Comment:
Use value: buildDollarDisplayItem(tvl)

And also, add the APY as another stats item:

{
  label: 'APY',
  value: buildPercentageDisplayitem(apy),
}